### PR TITLE
fix: return empty object when `tos` status is not present

### PR DESCRIPTION
**Pull request by mwwoda**
return empty object when tos status is not present
handle 201 code correctly in `setUserStatus`

### CODE REVIEW

File: src/managers/terms-of-service.ts [hunk 1]
```diff
@@ -274,7 +274,9 @@ class TermsOfService {
 				if (response.statusCode !== 200) {
 					throw errors.buildUnexpectedResponseError(response);
 				}
-				return response.body.entries[0];
+				return response.body.entries.length != 0
+					? response.body.entries[0]
+					: {};
```
Comment (mhagmajer) [inline src/managers/terms-of-service.ts:279]:
```suggestion
				return response.body.entries[0] ?? {};
```

**Comment by mhagmajer** [comment]
@mwwoda do you know what's up with the failing integration test?

**Comment by mwwoda** [comment]
Not quite sure about the reason of the failed test. Could just be a flaky one. After rerun it passes.

File: src/managers/terms-of-service.ts (hunk 1 above)
Comment (mwwoda) [reply src/managers/terms-of-service.ts]: fixed

**Review by mhagmajer — APPROVED** [review]
👍🏻

**Pull request merged**